feconf.py

```diff
@@ -33,6 +33,10 @@
 IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
 
 # Whether we should serve the development or production experience.
+# This is environment specific variable(DEV_MODE) changes to true only
```

Comment:
Ah, I see! OK, then add an additional comment:

When DEV_MODE is True, this indicates that we are not running in the production App Engine environment, which affects things like login/logout URLs, as well as third-party libraries that App Engine normally provides.